lib/tasks/avalon.rake

```diff
@@ -89,6 +89,23 @@ namespace :avalon do
         end
       end
     end
+
+    desc "Cleanup failed bookmarks"
+    task bookmark_cleanup: :environment do
+      Bookmark.all.each do |b|
+        if MediaObject.where(id: b.document_id).count > 0
+          puts "Bookmark pid: #{b.id} exists"
+        else
+          puts "Bookmark pid: #{b.id} does not exist. Deleting...."
+          begin
+            b.destroy
+          rescue Exception => e
+            puts "Failed to delete"
```

Comment:
I recommend output like this:

error message 1
error message 2
...
z bookmarks failed to deleted
y bookmarks deleted
x total bookmarks inspected